proposals/0242-alert-style-subtle.md

```diff
@@ -6,125 +6,203 @@
 * Impacted Platforms: [Core / iOS / Java Suite / HMI / RPC]
 
 ## Introduction
-This feature adds a new style of alert: the "subtle" alert, which does not take over the entire screen. The current alerts are moved to a style called "prominent."
+This feature adds the `SubtleAlert` RPC, which presents a style of alert: the "subtle" alert. This is a "notification style" alert that does not take over the entire screen.
 
 ## Motivation
 Competitive systems in this space often have "notification" style alerts, and user expectations are often set by mobile devices, which have notification alerts that don't take over the screen. To meet user and developer expectations, we should provide both prominent (modal) and subtle (notification) style alerts.
 
-Places where subtle alerts may be useful include messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presented outside of the current screen-takeover method.
+Subtle Alerts would be especially useful when the app is in the background. Places where subtle alerts may be useful include messaging apps that notify the user of a new message, a VOIP app notifying the user of an incoming call, or a navigation app notifying the user of an upcoming accident. These should have the ability to be presented outside of the current screen-takeover method.
 
 ## Proposed solution
-The proposed solution is to create a new `AlertStyle` enum that can be sent along with the `Alert` RPC.
+The proposed solution is to create a new RPC `SubtleAlert` that can be sent by developers to show a less prominent kind of Alert.
 
 ### Functionality
-Functionally, this would be identical to the prominent alert style with the following exceptions:
+Functionally, this would be identical to the standard (prominent / modal) `Alert` style with the following exceptions:
 
-1. It may support a different number of text lines or soft buttons.
-2. `progressIndicator` parameter should do nothing.
-3. Touching (or otherwise selecting) the alert should open the app.
-4. Touching outside of the subtle alert should close the alert.
+### MOBILE_API Changes
 
-### MOBILE_API and HMI_API Changes
-
-#### Capabilities
+#### Capabilities Updates
 ```xml
-<struct name="DisplayCapabilities" since="1.0">
-    <!-- All existing params -->
+<enum name="TextFieldName" since="1.0">
+    <!-- New Values -->
+    <element name="subtleAlertText1">
+        <description>The first line of the subtle alert text field; applies to `SubtleAlert` `alertText1`</description>
+    </element>
+    <element name="subtleAlertText2">
+        <description>The second line of the subtle alert text field; applies to `SubtleAlert` `alertText2`</description>
+    </element>
+    <element name="subtleAlertSoftButtonText">
```

Comment:
I don't see any other soft button text fields in the TextfieldName. Is this enum necessary?

Reply:
The reason I did it is because some situations may have constrained soft buttons. For example, widgets only permits images in soft buttons if I remember correctly. Since this is a notification, I thought that it might be constrained in a similar way where only images are allowed, or the text length is shorter than the rest of the system.